.babelrc

```diff
@@ -1,4 +1,7 @@
 {
-  "presets": ["es2015"],
-  "plugins": ["transform-runtime"]
+    "presets": ["es2015"],
```

Comment:
Add a comment here ([this is _JSON5_](https://babeljs.io/docs/usage/babelrc/) yay!) that this file is only used when transpiling src/ to lib/ in the prepublish phase, and should be kept in sync with the babel-loader configuration (`query`) in webpack.config.js?